src/libraries/System.Numerics.Vectors/ref/System.Numerics.Vectors.cs

```diff
@@ -269,32 +269,32 @@ public static partial class Vector
         public static System.Numerics.Vector<T> Multiply<T>(System.Numerics.Vector<T> left, System.Numerics.Vector<T> right) where T : struct { throw null; }
         public static System.Numerics.Vector<T> Multiply<T>(System.Numerics.Vector<T> left, T right) where T : struct { throw null; }
         public static System.Numerics.Vector<T> Multiply<T>(T left, System.Numerics.Vector<T> right) where T : struct { throw null; }
-        public static System.Numerics.Vector<System.Single> Narrow(System.Numerics.Vector<System.Double> source1, System.Numerics.Vector<System.Double> source2) { throw null; }
+        public static System.Numerics.Vector<System.Single> Narrow(System.Numerics.Vector<System.Double> low, System.Numerics.Vector<System.Double> high) { throw null; }
```

Comment:
cc: @tannergooding

Reply:
I'm fine with this as long as we are fine with the breaking change.

That being said, `lower` and `upper` may be better names than `low` and `high` here and would match the names we are using in the newer hardware intrinsics.

Reply:
I don't think we should be revisiting names at this point.  We should choose either src or ref as the winners.  So far we've felt that choosing src as the winner is slightly less breaking since a compile time break is easier to deal with than runtime break.